cfme/cloud/provider/__init__.py

```diff
@@ -53,9 +53,10 @@ class CloudProviderInstancesView(BaseLoggedInPage):
     """
     @property
     def is_displayed(self):
-        return (
-            self.breadcrumb.locations[0] == 'Cloud Providers' and
-            self.entities.title.text == '{} (All Instances)'.format(self.context['object'].name))
+        return (self.breadcrumb.locations[-1] == '{} (All Instances)'.format(
```

Comment:
I would prefer to see `expected_title = '{} (All Instances)'.format(self.context['object'].name)`, instead of two separate constructions of the string.

Reply:
fixed